src/Microsoft.DotNet.Interactive.Tests/CompositeKernelTests.cs

```diff
@@ -63,7 +64,82 @@ public async Task Handling_kernel_can_be_specified_using_kernel_name_as_a_direct
             events.Should()
                   .ContainSingle<CommandHandled>(e => e.Command == fsharpCommand);
         }
-        
+
+        [FactSkipLinux]
+        public async Task Handling_kernel_can_be_specified_using_kernel_name_as_a_directive_as_a_proxy_named_pipe()
```

Comment:
I'd move these tests out of `CompositeKernelTests`.

Reply:
Ok, added ProxyKernelTests